Azrux/Athenas-front
# Add config to use NextUI

## Description
Installed NextUI, now you can use these components in the project: 
- https://storybook.nextui.org/?path=/docs/welcome--docs

## How to test?
- Run `npm i`
- Run `npm run dev`
- Go to `localhost:3000`
- There you can a see a button
- Go to `src/pages/landing` and try to import any ohter component you want
